public/components/page-contributors/contributors/contributors.stache

```diff
@@ -10,24 +10,24 @@
     </tr>
   </thead>
   <tbody>
-  {{#each contributors as contributor}}
+  {{# each(contributors, contributor=value) }}
     <tr>
-      <td><model-edit-property model:from="contributor" property="name" /></td>
-      <td><model-edit-property model:from="contributor" property="email" /></td>
+      <td><model-edit-property model:from="contributor" property:from="'name'" /></td>
```

Comment:
"normal" looking attributes are not setup as `can-stache-bindings` anymore.  Instead, you must use `:from` or `:raw` to setup a binding.